app/src/parser_print_protobuf.c

```diff
@@ -745,6 +745,22 @@ static parser_error_t parser_getItemFollow(uint8_t displayIdx,
                 snprintf(outVal, outValLen, "Node Provider Rewards");
                 return parser_ok;
             }
+            case ic_nns_governance_pb_v1_Topic_TOPIC_SNS_DECENTRALIZATION_SALE : {
+                snprintf(outVal, outValLen, "SNS Decentralization Swap");
+                return parser_ok;
+            }
+            case ic_nns_governance_pb_v1_Topic_TOPIC_SUBNET_REPLICA_VERSION_MANAGEMENT : {
+                snprintf(outVal, outValLen, "Subnet Replica Version Management");
```

Comment:
Do this fit on the screen? Shouldn't we replace **snprint** with **pageString** function?

Reply:
It fits!